package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "five-bells-notary",
-  "version": "2.0.2",
```

Comment:
Since this is an API change (PUT to POST), should we update the major version number? So: 3.0.0.